XML/TestCases/FunctionalTests-KernelDebug.xml

```diff
@@ -0,0 +1,15 @@
+<TestCases>
+    <test>
+        <testName>KERNEL-DEBUG-BASIC</testName>
```

Comment:
I suggest you rename this test case with RHEL-, so that no one get confused about it's purpose, when skipped for other distro.

Reply:
> > @harz566 Do you have more suggestions for this pull request? For "Main -VMname xxx", how about keeping current style?
> > Thank you so much.
> 
> Hi @lixuemin2016 , it's OK, but it looks not neat/simple in coding, and by the way, you declared '-TestParams $TestParams' (which should be a semicolon-separated string or empty), but never used in Main. Let me clarify: you can leave the bottom line as simple as it is: Main.
> Then at top of Main, remove the param() section, and inside of Main(), you can replace '$VMname' directly with $AllVMData.RoleName, replace '$Ipv4' directly with $AllVMData.PublicIP, replace '$VMPort' directly with $AllVMData.SSHPort, replace '$VMUserName' directly with $global:user, and replace $VMPassword directly with $global:password. That's my suggested changes.

@harz566, Got it, thank you so much for your detailed explanation.
If update as above, the Main looks very clean. 

Just want to reach agreement with you, for current powershell scripts's main function in LISv2 repo, there are multiple styles, here lists three types:
1. Main -TestParams (ConvertFrom-StringData $TestParams.Replace(";","`n")) -AllVMData $AllVMData   (when use Testparams) 
2. Main  (empty, for example nested feature related script )
3. Main -VMName $AllVMData.RoleName -hvServer $GlobalConfig.Global.Hyperv.Hosts.ChildNodes[0].ServerName `
         -ipv4 $AllVMData.PublicIP -VMPort $AllVMData.SSHPort `
         -VMUserName $user -VMPassword $password -rootDir $WorkingDirectory `
         -testParams $testParams  (for example, most of scripts from old lis-test repo, KVP, storage related)

Would you prefer to use $AllVMData.RoleName, $AllVMData.PublicIP, $AllVMData.SSHPort, $global:user, $global:password in the powershell scripts, yes?  Just want to keep same style with all of your scripts in future, not only for this kernel-debug script.

Thank you so much.
@haiyangz @LiliDeng @juhlee-microsoft